support-models/src/main/scala/com/gu/support/catalog/Product.scala

```diff
@@ -212,224 +218,73 @@ case object Paper extends Product {
+    id, billingPeriod, Domestic, NoProductOptions, description,
+    productRatePlanChargeId = productRatePlanChargeId, readerType = readerType
+  )
+
+  private def restOfWorld(
+    id: String,
+    billingPeriod:
+    BillingPeriod,
+    description: String,
+    productRatePlanChargeId: Option[String] = None,
+    readerType: ReaderType = Direct
+  ) = ProductRatePlan(
+    id, billingPeriod, RestOfWorld, NoProductOptions, description,
     productRatePlanChargeId = productRatePlanChargeId, readerType = readerType
   )
 
   lazy val ratePlans: Map[TouchPointEnvironment, List[ProductRatePlan[GuardianWeekly.type]]] =
     Map(
       PROD -> List(
-        productRatePlan(
-          "2c92a0086619bf8901661ab545f51b21",
-          SixWeekly,
-          RestOfWorld,
-          "Guardian Weekly 6 for 6, rest of world delivery",
-          productRatePlanChargeId = Some("2c92a0086619bf8901661ab546091b23")
-        ),
-        productRatePlan(
-          "2c92a0fe6619b4b601661ab300222651",
-          Annual,
-          RestOfWorld,
-          "Guardian Weekly annual, rest of world delivery"
-        ),
-        productRatePlan(
-          "2c92a0ff67cebd140167f0a2f66a12eb",
-          Annual,
-          RestOfWorld,
-          "Guardian Weekly one year, rest of world delivery",
-          readerType = Gift
-        ),
-        productRatePlan(
-          "2c92a0086619bf8901661ab02752722f",
-          Quarterly,
-          RestOfWorld,
-          "Guardian Weekly quarterly, rest of world delivery"
-        ),
-        productRatePlan(
-          "2c92a0076dd9892e016df8503e7c6c48",
-          Quarterly,
-          RestOfWorld,
-          "Guardian Weekly three month, rest of world delivery",
-          readerType = Gift
-        ),
-        productRatePlan(
-          "2c92a0086619bf8901661aaac94257fe",
-          SixWeekly,
-          Domestic,
-          "Guardian Weekly 6 for 6, domestic delivery",
-          productRatePlanChargeId = Some("2c92a0086619bf8901661aaac95d5800")
-        ),
-        productRatePlan(
-          "2c92a0fe6619b4b901661aa8e66c1692",
-          Annual,
-          Domestic,
-          "Guardian Weekly annual, domestic delivery"
-        ),
-        productRatePlan(
-          "2c92a0ff67cebd0d0167f0a1a834234e",
-          Annual,
-          Domestic,
-          "Guardian Weekly one year, domestic delivery",
-          readerType = Gift
-        ),
-        productRatePlan(
-          "2c92a0fe6619b4b301661aa494392ee2",
-          Quarterly,
-          Domestic,
-          "Guardian Weekly quarterly, domestic delivery"
-        ),
-        productRatePlan(
-          "2c92a00e6dd988e2016df85387417498",
-          Quarterly,
-          Domestic,
-          "Guardian Weekly three months, domestic delivery",
-          readerType = Gift)
-
+        restOfWorld("2c92a0086619bf8901661ab545f51b21", SixWeekly, "Guardian Weekly 6 for 6, rest of world delivery",
```

Comment:
I've moved these back to one per line because I was finding it very hard to work with in the previous format, the list was too long to scan easily. To reduce the number of params on each line I've added a new helper method above so that it is only Six for Six plans which break onto two lines

Reply:
it's a tricky balance to tabulate complicated data, makes sense to keep it under review